.travis.yml

```diff
@@ -53,6 +58,22 @@ matrix:
       env: [ ARCH=x86_64, BUILD_TYPE=Debug, GENERATOR="Visual Studio 15 2017" ]
     - <<: *windows_vs2017
       env: [ ARCH=x86_64, BUILD_TYPE=Release, GENERATOR="Visual Studio 15 2017" ]
+    - stage: Coverity
+      <<: *linux_gcc8
+      name: Coverity Build
+      env:
+        - BUILD_TYPE=Release
+        - C_COMPILER=gcc-8
+        - CXX_COMPILER=g++-8
+        - USE_SANITIZER=none
+        - COVERITY_SCAN_PROJECT_NAME="$TRAVIS_REPO_SLUG"
+        - COVERITY_SCAN_BRANCH_PATTERN="coverity_scan"
```

Comment:
I prefer this would just run on master. We don't have a coverity_scan branch and I doubt we're willing to go through the trouble of creating one, when it's just as easy to just run it on master.